public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -18,46 +18,11 @@ import EpicTestTargetContentEditor from './epicTestTargetContentEditor';
 import EpicTestMaxViewsEditor from './epicTestMaxViewsEditor';
 import useValidation from '../hooks/useValidation';
 import { articleCountTemplate, countryNameTemplate } from '../helpers/copyTemplates';
-import EpicTestVariantsSplitEditor from './epicTestVariantsSplitEditor';
+import TestVariantsSplitEditor from '../testVariantsSplitEditor';
 import { getDefaultVariant } from './utils/defaults';
 import LiveSwitch from '../../shared/liveSwitch';
-
-const useStyles = makeStyles(({ spacing, palette }: Theme) => ({
-  container: {
-    width: '100%',
-    height: 'max-content',
-    background: '#FFFFFF',
-    paddingTop: spacing(6),
-    paddingRight: spacing(12),
-    paddingLeft: spacing(3),
-  },
-  headerAndSwitchContainer: {
-    paddingBottom: spacing(3),
-    borderBottom: `1px solid ${palette.grey[500]}`,
-
-    '& > * + *': {
-      marginTop: spacing(2),
-    },
-  },
-  sectionContainer: {
-    paddingTop: spacing(1),
-    paddingBottom: spacing(6),
-    borderBottom: `1px solid ${palette.grey[500]}`,
-
-    '& > * + *': {
-      marginTop: spacing(4),
-    },
-  },
-  sectionHeader: {
-    fontSize: 16,
-    fontWeight: 500,
-    color: palette.grey[700],
-  },
-  buttonsContainer: {
-    paddingTop: spacing(4),
-    paddingBottom: spacing(12),
-  },
-}));
```

Comment:
I think it's great to share styles! But the `useStyles` approach is actually the newer, preferred method. Really we should refactor all the components to use `useStyles` over the `withStyles` approach but we haven't gotten around to it! I'd say we'd be better off updating the `bannerTestEditor` instead of this one.